src/components/ContentPreview/Header.js

```diff
@@ -17,85 +19,127 @@ import ContentOpenWith from '../ContentOpenWith/ContentOpenWith';
     const closeMsg = intl.formatMessage(messages.close);
     const printMsg = intl.formatMessage(messages.print);
     const downloadMsg = intl.formatMessage(messages.download);
+    const drawMsg = intl.formatMessage(messages.drawAnnotation);
+    const pointMsg = intl.formatMessage(messages.pointAnnotation);
     const shouldRenderOpenWith = id && contentOpenWithProps.show;
 
     return (
         <div className="bcpr-header">
-            <div className="bcpr-name">
-                {file ? getIcon(24, file) : null}
-                <span>{name}</span>
-            </div>
-            <div className="bcpr-btns">
-                {shouldRenderOpenWith && (
-                    <ContentOpenWith
-                        className="bcpr-bcow-btn"
-                        fileId={id}
-                        token={token}
-                        {...contentOpenWithProps}
-                    />
-                )}
-                {canDownload && (
-                    <PlainButton
-                        type="button"
-                        className="bcpr-btn"
-                        onClick={onPrint}
-                        title={printMsg}
-                        aria-label={printMsg}
-                    >
-                        <IconPrint color={COLOR_999} width={22} height={22} />
-                    </PlainButton>
-                )}
-                {canDownload && (
-                    <PlainButton
-                        type="button"
-                        className="bcpr-btn"
-                        onClick={onDownload}
-                        title={downloadMsg}
-                        aria-label={downloadMsg}
-                    >
-                        <IconDownload
-                            color={COLOR_999}
-                            width={18}
-                            height={18}
+            <div className="bp-header bp-base-header">
+                <div className="bcpr-name">
+                    {file ? getIcon(24, file) : null}
+                    <span>{name}</span>
+                </div>
+                <div className="bcpr-btns">
+                    {shouldRenderOpenWith && (
+                        <ContentOpenWith
+                            className="bcpr-bcow-btn"
+                            fileId={id}
+                            token={token}
+                            {...contentOpenWithProps}
                         />
-                    </PlainButton>
-                )}
-                {onClose && (
-                    <PlainButton
-                        type="button"
-                        className="bcpr-btn"
-                        onClick={onClose}
-                        title={closeMsg}
-                        aria-label={closeMsg}
-                    >
-                        <IconClose color={COLOR_999} width={24} height={24} />
-                    </PlainButton>
-                )}
+                    )}
+                    {canAnnotate && (
```

Comment:
Would this toggle appropriately if the user has disabled a particular annotations type? https://github.com/box/box-annotations/blob/master/docs/enabling-types.md

Reply:
yup, it looks like it does. Should work the same as in preview since we're relying on Box Annotations to do the hiding and showing for us.